spec/integration/account_migration_spec.rb

```diff
@@ -180,8 +186,10 @@ def run_migration
 
     # this is the case when a user changes diaspora id but stays on the same pod
     context "old user is local and new user is local" do
-      include_context "with local old user"
-      include_context "with local new user"
+      let(:old_user) { FactoryGirl.create(:user) }
+      let(:old_person) { old_user.person }
+      let(:new_user) { FactoryGirl.create(:user) }
+      let(:new_person) { new_user.person }
 
       def run_migration
         AccountMigration.create!(old_person: old_user.person, new_person: new_user.person).perform!
```

Comment:
`old_user.person` can be simplified to `old_person`, and same for `new_...` (and at "old user is remote and new user is local" above too).